pkg/term/windows/windows.go

```diff
@@ -3,3 +3,27 @@
 // and return pseudo-streams that convert ANSI sequences to / from Windows Console API calls.
 
 package windows
+
+import (
+	"io/ioutil"
+	"os"
+
+	ansiterm "github.com/Azure/go-ansiterm"
+	"github.com/Sirupsen/logrus"
+)
+
+var logger *logrus.Logger
+
+func init() {
+	logFile := ioutil.Discard
+
+	if isDebugEnv := os.Getenv(ansiterm.LogEnv); isDebugEnv == "1" {
+		logFile, _ = os.Create("ansiReaderWriter.log")
```

Comment:
Spoke to @jstarks in person. We are guaranteed that there is no synchronisation issue here - the same thread initialises both the reader and writer. There's no need to not use a pattern which is in dozens of places in the docker codebase (ie `init`). Unless I'm missing something else? :innocent:  :confused: